test/forge/libraries/periphery/MorphoBalancesLibTest.sol

```diff
@@ -73,31 +73,31 @@ contract MorphoBalancesLibTest is BaseTest {
     {
         _generatePendingInterest(amountSupplied, amountBorrowed, timeElapsed, fee);
 
-        uint256 expectedSupplyBalance = morpho.expectedSupplyBalance(marketParams, address(this));
+        uint256 expectedSupplyAssets = morpho.expectedSupplyAssets(marketParams, address(this));
```

Comment:
```suggestion
        uint256 expectedSupplyBalance = morpho.expectedSupplyAssets(marketParams, address(this));
```
so that we keep the same naming for the expected value and the actual value. I don't think that having balance is a problem here, the naming "balance" is used liberally in a lot of places and it's fine

Reply:
> Perhaps we should update the test name instead?

It wouldn't help with the duality expected/actual. In any case this is not a blocker for me